packages/service/package.json

```diff
@@ -31,6 +31,9 @@
     "y-websocket": "^1.3.15",
     "yjs": "^13.5.10"
   },
+  "prisma": {
+    "schema": "../../db/schema.prisma"
+  },
```

Comment:
... does this still make sense?

Reply:
I had that changed, but I think I lost it along the way when structuring commits somehow. Will fix this now